src/services/calendarEventService.ts

```diff
@@ -220,3 +220,57 @@ function parseUserEventsQueryResult(
 ): CalendarEvent & { price: string } {
   return { ...parseQueryResult(row), price: row.price as string }
 }
+
+export async function createCalendarEvent(
+  props: {
+    name: string, // Looks weird on absence
+    user_id: number | null,
+    created: Date | null,
+    starts: Date, // members.tko-aly.fi throws error on absence
+    registration_starts: Date | null,
+    registration_ends: Date | null, // If registration_starts is given but not _ends, members throws error
+    cancellation_starts: Date | null,
+    cancellation_ends: Date | null,
+    location: string | null,
+    category: string | null,
+    description: string | null,
+    price: string | null,
+    map: string | null,
+    membership_required: boolean | null,
+    outsiders_allowed: boolean | null,
+    template: boolean, // members.tko-aly.fi won't render on absence
+    responsible: string | null,
+    show_responsible: boolean | null,
+    avec: boolean | null,
+    deleted: boolean, // members.tko-aly.fi won't render on absence
+    alcohol_meter: number | null,
+  }
+): Promise<knex.Knex.QueryBuilder<any, number[]>> {
+  const calendarEvent = {
```

Comment:
Jos `props`-muuttujan sisältö tarkastetaan jo pyynnön käsittelijässä, kuten on hyvän käytännön mukaista, ei tässä kohta enään tarvitse tehdä tällaista, vaan tiedot voi antaa suoraan eteenpäin Knexille.

Reply:
tää on muute erinomaisen hyvä pointti :D